SparkPost/2web2ui
# FE-1148 - [Domains UX Redesign] List page - Tabs Setup

[FE-1148](https://jira.int.messagesystems.com/browse/FE-1148)

### What Changed
- Added tabs to the new domains page that render the following routes when clicked:
  + `/domains/list/sending`
  + `/domains/list/bounce`
  + `/domains//list/tracking`
- Updates route config for domains details pages

### How To Test
1. Ensure the `allow_domains_v2` is enabled, ensure Hibana is enabled, and visit `/domains`
2. Visit `/domains` and ensure you are redirected to `/domains/list/sending`
3. Visit `/domains/list` and ensure you are redirected to `/domains/list/sending`
4. Click on the tabs and verify the correct content/routes render

### To Do
